en/docs/Learn/Tutorials/create-a-websocket-api.md

```diff
@@ -0,0 +1,82 @@
+
+In this tutorial, you create and publish an API with a WebSocket backend and then invoke it using a Netty-based WebSocket client. You can use any WebSocket client to invoke the API.
+
+1. Sign in to the WSO2 API Publisher providing username and password `https://<hostname>:9443/publisher` (e.g., `https://localhost:9443/publisher` ).
+    <html><div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>CREATE button will only appear for user who has creator role permission.</p>
+      </div>
+    </html>
+
+2.  Go to **CREATE API** and Click **Design a New Websocket API**.
+    ![](../../../../assets/img/Learn/create-websocket-api.jpg)
+
+3.  Provide the information in the table below and click **CREATE** or **CREATE & PUBLISH**.
+
+    | Field   | Sample value   |
+    |---------|----------------|
+    | Name    | EchoWebSocket  |
+    | Context | /echowebsocket |
+    | Version | 1.0            |
+    | Endpoint| ws://echo.websocket.org:80|
+    | Business Paln | Gold,Silver|
+    
+    ![](../../../../assets/img/Learn/create-web-socket-api.jpg)
+
+    <html>
+     <div class="admonition note">
+     <p class="admonition-title">Note</p>
+     <p>The **CREATE & PUBLISH** option will appear only if the optional fields **Endpoint** and **Business plan(s)** are provided by a user who has publisher permission.</p>
+     </div>
+     </html>
+
+5.  When click **CREATE** or **CREATE & PUBLISH**, page will be redirected to the overview page of the created WebSocket API.
+    ![](../../../../assets/img/Learn/overview-websocket-api.jpg)
+
+6.  Navigate to Endpoint page, check the endpoints and click following to add details if needed.
+    ![](../../../../assets/img/Learn/endpoint-view-of-websocket-api.jpg)
+
+Now, the WebSocket API created and configured successfully. Click **PUBLISH** 
+   You have now published the WebSocket API to the API developer portal. Let's subscribe to it.
+
+8.  When prompted, choose to open the newly published API in the API Store.
+9.  The `EchoWebSocket` API opens. Select an application (e.g., DefaultApplication), the **Gold tier** and subscribe to the API.
+    ![]({{base_path}}/assets/attachments/103328746/103328730.png)
+10. Click the **View Subscriptions** button when prompted. The **Subscriptions** tab opens.
+
+11. Click the **Production Keys** tab and click **Generate Keys** to create an application access token. If you have already generated keys before, click **Regenerate** .
+    ![]({{base_path}}/assets/attachments/103328746/103328729.png)
+    You can also add a [Callback URL](_Key_Concepts_) , if you have not added it already when creating the API. You have now subscribed to an API in the API Store and can invoke it using a WebSocket client. In this tutorial, you invoke it using a [Netty-based WebSocket client](https://docs.wso2.com/download/attachments/92520686/sample-ws-client.zip?version=1&modificationDate=1484568275000&api=v2) .
```

Comment:
here as well

Reply:
This is because we moved the invocation part of the WebSocket API to the tutorial session. Still, the tutorial part is in progress. Will do the changes with it.